examples/ts-without-babel/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "ts-without-babel",
+  "version": "0.1.0",
+  "author": "Eugene Scherbinin",
```

Comment:
I was thinking - maybe we should remove the author/license field from these examples, seems like that makes more sense ...

Reply:
I haven't found any info on requirement of author and license. `npm init -y` creates both, and author is empty. I removed them in my example and tested  - okay.